connect/eaas/core/deployment/helpers.py

```
@@ -0,0 +1,92 @@
+import re
+import subprocess
+from collections import OrderedDict
+from distutils.version import StrictVersion
+
+from connect.eaas.core.validation.models import ValidationItem
+
+
+DEFAULT_CLONE_DIR = '_repo'
+
+
+def clone_repo(temp_path, repo_url):
+    result = subprocess.run(
+        [
+            'git',
+            '-C',
+            temp_path,
+            'clone',
+            repo_url,
+            DEFAULT_CLONE_DIR,
+        ],
+        capture_output=True,
+        stdin=subprocess.DEVNULL,
+        start_new_session=True,
+    )
+    try:
+        result.check_returncode()
+    except subprocess.CalledProcessError:
+        return ValidationItem(
```

I would suggest to raise an exception instead of returning a ValidationItem